Cargo.toml

```diff
@@ -73,6 +74,5 @@ required-features = ["ctaphid", "devel"]
 
 [patch.crates-io]
 flexiber = { git = "https://github.com/Nitrokey/flexiber", tag = "0.1.1.nitrokey" }
-trussed = { git = "https://github.com/Nitrokey/trussed", rev = "52da9bb" }
-littlefs2 = { git =  "https://github.com/Nitrokey/littlefs2", tag = "v0.3.2-nitrokey-2" }
-trussed-auth = { git = "https://github.com/trussed-dev/trussed-auth/", rev = "6beabd125ea57e87bf642e18278fcc2ba224c50c" }
\ No newline at end of file
+trussed = { git = "https://github.com/Nitrokey/trussed", tag = "v0.1.0-nitrokey.11" }
+trussed-auth = { git = "https://github.com/trussed-dev/trussed-auth/", tag = "v0.2.2" }
```

Comment:
There should be a tag for fido-authenticator too.

Reply:
I have used a tag. Do you mean having an usual crate.io dependency, and a git dep as patch? It does not want to work that way sadly. Do you know perhaps how can I make it work?

Reply:
Ah, sorry, my bad. I have not sent that change yet. Updated.